### PR TITLE
docs: translated Fragments.md

### PR DESCRIPTION
**Pull request by RickyWooo**
Follow the existing translation pattern and translate Fragments.md, feel free to let me know if there is any problem, thank you.

### CODE REVIEW

**Comment by RickyWooo** [comment]
Thanks @bugtender & @neighborhood999 for both of your helps, I will review it carefully and create another PR.

**Comment by neighborhood999** [comment]
Hi @RickyWooo, you don't need to close this PR, just edit and push it.

**Comment by RickyWooo** [comment]
@neighborhood999 I see :) it just my working habits :joy:, I have modified on the documents for your reference, thank you.

File: content/docs/fragments.md [hunk 1]
```diff
@@ -110,12 +111,11 @@ class Columns extends React.Component {
...
 }
 ```
-
-You can use `<></>` the same way you'd use any other element except that it doesn't support keys or attributes.
+你可以像使用其他元素一樣使用 `<></>`，但值得注意的是它並不支援 key 和 attribute。
 
 ### Keyed Fragments {#keyed-fragments}
 
-Fragments declared with the explicit `<React.Fragment>` syntax may have keys. A use case for this is mapping a collection to an array of fragments -- for example, to create a description list:
+透過明確宣告 `<React.Fragment>` 的 Fragment 可能會遇到帶有 key 的情況。一個使用案例是將它 mapping 到 Fragment array。舉例來說，像下方程式碼一樣建立一個敘述列表。
```
[the remark this answers is not in the document]
Comment (neighborhood999) [reply content/docs/fragments.md]: Below code comment need to translate:

```diff
+ // Without the `key`, React will fire a key warning
```

File: content/docs/fragments.md [hunk 2]
```diff
@@ -133,8 +133,8 @@ function Glossary(props) {
 }
 ```
 
-`key` is the only attribute that can be passed to `Fragment`. In the future, we may add support for additional attributes, such as event handlers.
+目前 `key` 是唯一可以傳遞給 `Fragment` 的 attribute 。之後我們可能會支援更多的 attribute，像是 event handlers。
```
Comment (neighborhood999) [inline content/docs/fragments.md]:
```suggestion
目前 `key` 是唯一可以傳遞給 `Fragment` 的 attribute。之後我們可能會支援更多的 attribute，像是 event handler。
```

File: content/docs/fragments.md [hunk 3]
```diff
@@ -133,8 +133,8 @@ function Glossary(props) {
...
 
-### Live Demo {#live-demo}
+### 線上展示 {#live-demo}
```
Comment (neighborhood999) [inline content/docs/fragments.md]: I prefer to keep original in here.

File: content/docs/fragments.md [hunk 4]
```diff
@@ -94,9 +95,9 @@ which results in a correct `<Table />` output of:
 </table>
 ```
 
-### Short Syntax {#short-syntax}
+### 更簡短的語法 {#short-syntax}
```
[the remark this answers is not in the document]
Comment (RickyWooo) [reply content/docs/fragments.md]: sure

**Comment by RickyWooo** [comment]
Thanks for @neighborhood999's effort,  I have push the latest translation :)

**Pull request merged**